masonite/provider.py

```diff
@@ -38,3 +41,40 @@ def load_app(self, app):
 
         self.app = app
         return self
+
+    def routes(self, routes):
+        web_routes = self.app.make('WebRoutes')
+        web_routes += routes
+
+    def http_middleware(self, middleware):
+        http_middleware = self.app.make('HttpMiddleware')
+        http_middleware += middleware
+
+    def route_middleware(self, middleware):
+        route_middleware = self.app.make('RouteMiddleware')
+        route_middleware.update(middleware)
+
+    def migrations(self, *directories):
+
+        for directory in directories:
+            random_string = ''.join(random.choice(
```

Comment:
I think this could be refactored in a helper method like this

```python

import hashlib
import uuid
import string
import random


def random_string(length=4):
    rnd_string = "".join(
        random.choice(string.ascii_uppercase + string.digits) for _ in range(length)
    )
    return rnd_string


def random_hash():
    uid = uuid.uuid4().bytes
    return hashlib.sha256(uid).hexdigest()


print(random_string())

print(random_hash())

```